### PR TITLE
MoMMIs can scan phazon once again

### LINKED ISSUE #16731: Mommi material synthesizer
Description of issue

Mommis material synthesizer cant scan bananium and phazon.

Steps to reproduce

Try to scan bananium or phazon as a mommi.

What you Expected

To have the power to scan phazon and bananium.

#16627

### PR DESCRIPTION
**Pull request by ghost**
Fixes #16731
:cl:
 * bugfix: MoMMIs can scan phazon.

### CODE REVIEW

File: code/game/objects/items/devices/mat_synth.dm [hunk 1]
```diff
@@ -34,9 +34,10 @@
 							 "floor tiles" = /obj/item/stack/tile/plasteel,
 							 "metal rods" = /obj/item/stack/rods)
 
-/obj/item/device/material_synth/robot/New()
+/obj/item/device/material_synth/robot/New() //We have to do this during New() because BYOND can't pull a typesof() during compile time.
 	. = ..()
-	can_scan = existing_typesof(/obj/item/stack/sheet) - list(/obj/item/stack/sheet/mineral/clown, /obj/item/stack/sheet/mineral/phazon)
+	if(!istype(src, /obj/item/device/material_synth/robot/mommi))
```
Comment (Exxion) [inline code/game/objects/items/devices/mat_synth.dm]: lrn2OOP

(Just re-add them in the MoMMI one's `New()`, or add a string deal like storage items, or something)

**Review by Exxion — DISMISSED** [review]

File: code/game/objects/items/devices/mat_synth.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/devices/mat_synth.dm]: I had it working like that at first, but didn't like that the "non-borg can scan list" was copypasted twice that way.

**Comment by SonixApache** [comment]
@Unknown-as-Captain 
put a descriptive name in the name field and `fixes #xxxx` in the comment field

**Comment by gbasood** [comment]
changelog it up because why not get your valids

**Comment by ghost** [comment]
The changelog already says this is how it should work

The change is 3 lines, what needs explaining?

**Comment by SonixApache** [comment]
there is no changelog, only commit titles
just follow the gosh darn standards

**Comment by gbasood** [comment]
literally just put this in the op
:cl:
 - bugfix: fixes that bug

**Comment by ghost** [comment]
@SonixApache the previous changelog
I also went and checked, out of the last 25 bugfixes merged, 20 didn't have a changelog, and CONTRIBUTING.md doesn't say anything about that, so if it's a standard it's not a very good one

**Pull request merged**